BLaurenB/techtime
# User story 16

#### What's this PR do?
This revisits User Story 16. We made sure that users are only able to view their own orders.
#### Where should the reviewer start- what are the pertinent changes and the sequence?
The user controller was updated, as well as the tests
#### How should this be manually tested?
check order pages and manually type in an order id to see if you can access another user's stuff.
#### Any background context you want to provide?
#### What are the relevant tickets?
Closes #72 Closes #94 
#### Screenshots (if appropriate)
#### Questions:
